### PR TITLE
[PIL-665] Copy Content Updates to existing screens (Part 2)

### PR DESCRIPTION
**Pull request by aashik-varahan**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -623,12 +623,11 @@ subscriptionAddress.heading = What address do you want to use as the filing memb
...
 subscriptionAddress.addressLine1 = Address line 1
 subscriptionAddress.addressLine2 = Address line 2 (optional)
-subscriptionAddress.town_city = Town or City
+subscriptionAddress.town_city = Town or city
 subscriptionAddress.region = Region (optional)
 subscriptionAddress.postcode = Postal code (if applicable)
-subscriptionAddress.postcode.hint = For a UK address, you must enter a correctly formatted UK postcode
 subscriptionAddress.country = Country
-subscriptionAddress.country.hint = Enter text and then choose from the list
+subscriptionAddress.country.hint = Enter text and then choose from the list.
```
Comment (jassalrichy) [inline conf/messages.en:630]: Need to remove duplicates for:
subscriptionAddress.country
subscriptionAddress.country.hint

Comment (aashik-varahan) [reply conf/messages.en:630]: subscriptionAddress content changes are part of another ticket PIL-663

**Review by jassalrichy — APPROVED** [review]

File: conf/messages.en (hunk 1 above)
Comment (aashik-varahan) [reply conf/messages.en:630]: Removed the duplicate entry for 
subscriptionAddress.country.hint
subscriptionAddress.country
subscriptionAddress.checkYourAnswersLabel

File: conf/messages.en [hunk 2]
```diff
@@ -565,12 +565,12 @@ checkYourAnswers.nfm = Nominated filing member
 
 addSecondaryContact.title = Is there someone else we can contact if {0} is not available?
 addSecondaryContact.heading = Is there someone else we can contact if {0} is not available?
-addSecondaryContact.hint = This can be a team mailbox or another contact who is able to deal with enquiries about the group''s management of Pillar 2 top-up tax
+addSecondaryContact.hint = This can be a team mailbox or another contact who is able to deal with enquiries about the group''s management of Pillar 2 top-up taxes.
 addSecondaryContact.error.required = Select yes if there is someone else we can contact
```
Comment (jassalrichy) [inline conf/messages.en:569]: We should be using ‘smart’ (curly) apostrophes rather than 'dumb' (straight) apostrophes.

Comment (aashik-varahan) [reply conf/messages.en:577]: This has already been discussed and implemented in PIL-683

**Review by javed-iqbal1 — APPROVED** [review]

**Pull request merged**